### PR TITLE
Update weekly checklist template

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/weekly-checklist.md [hunk 1]
```diff
@@ -6,21 +6,28 @@ labels: ''
...
+    - [ ]  Send email to mentees
+    - [ ]  Send email to speakers
+2. Cohort call video preparation
+    - [ ]  Download from Zoom cloud recordings
+    - [ ]  Upload to [Otter.ai](http://otter.ai/) (*will need a few minutes to process*)
+    - [ ]  Upload to the OLS YouTube account
+    - [ ]  Create a front page slide for the video - templates are available in Google Drive
+    - [ ]  Make sure the YouTube video is added to the current cohort playlist
+    - [ ]  Correct [Otter.ai](http://otter.ai/) transcripts (*make sure to use uploaded video, not the original transcript, as the timings won't match the video*)
+    - [ ]  Download corrected transcript as a .srt file from Otter
```
Comment (da5nsy) [inline .github/ISSUE_TEMPLATE/weekly-checklist.md:21]: @emmyft - with 'include speaker names' and 'add line breaks automatically' checked?

Comment (emmyft) [reply .github/ISSUE_TEMPLATE/weekly-checklist.md:21]: hmm... I don't think so? the [hackmd](https://hackmd.io/@openlifesci/cohort-call-wranglers) has more details!

**Review by emmyft — APPROVED** [review]

**Comment by ricMbugua** [comment]
Hi can I work on this issue?

**Comment by yochannah** [comment]
this is being handled elsewhere by @iramosp and @Tajuddeen1

**Pull request closed without merging**